packages/cojson/src/sync.ts

```diff
@@ -667,6 +665,8 @@ export class SyncManager {
         console.error("Error sending known state", e);
       });
     }
+
+    await this.syncCoValue(coValue);
```

Comment:
We do send a correction/ack message before syncing to give an immediate response to the peers that are waiting for confirmation that a value is fully synced